db/schema.rb

```diff
@@ -21,11 +21,11 @@
   create_table "participants", force: :cascade do |t|
     t.integer  "project_id",     null: false
     t.string   "email",          null: false
-    t.string  "secret_id"
     t.datetime "last_responded"
+    t.string   "secret_id"
   end
 
-  add_index "participants", ["project_id", "email"], name: "index_participants_on_project_id_and_email", unique: true
+  add_index "participants", ["project_id", "email", "last_responded"], name: "index_participants_on_project_id_and_email_and_last_responded", unique: true
```

Comment:
this change was undone in Joyce's PR commit. fix merge conflicts carefully!
We don't want to index on all three columns as unique, only need to enforce uniqueness on project_id and email